tfjs-backend-cpu/src/backend_cpu.ts

```diff
@@ -3795,5 +3794,3 @@ export class MathBackendCPU extends KernelBackend {
     return buffer.toTensor().reshape(shape);
   }
 }
-
-ENGINE.registerBackend('cpu', () => new MathBackendCPU(), 1 /* priority */);
```

Comment:
Currently I think the WebGL backend assumes there's a registered CPU backend for CPU forwarding, something to be keep in mind when we pull out WebGL.